docs/src/pages/components/tables/MaterialTableDemo.js

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import MaterialTable from 'material-table';
+import Promise from 'core-js-pure/features/promise'; // required for IE 11
```

Comment:
Required for IE11. Crashes currently: https://material-ui.com/components/tables/

Reply:
I still get crashes on the table page

Reply:
Was about to mark this as ready. It worked fine with `docs:dev` locally. Will investigate.

Reply:
It's already polyfilled on the landing page. I couldn't find any documentation about what's already polyfilled by nextjs.